src/GovUk.Education.ExploreEducationStatistics.Data.Services.Tests/SubjectResultMetaServiceTests.cs

```diff
@@ -279,13 +286,13 @@ public async Task GetSubjectMeta_BoundaryLevelViewModelsReturnedForSubject()
 
             await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
             {
-                await contentDbContext.Publications.AddAsync(publication);
+                contentDbContext.Publications.Add(publication);
                 await contentDbContext.SaveChangesAsync();
             }
 
             await using (var statisticsDbContext = InMemoryStatisticsDbContext(statisticsDbContextId))
             {
-                await statisticsDbContext.ReleaseSubject.AddAsync(releaseSubject);
+                statisticsDbContext.ReleaseSubject.Add(releaseSubject);
                 await statisticsDbContext.SaveChangesAsync();
```

Comment:
My interpretation of our recent decision to prefer `Add/AddRange` over `AddAsync/AddRangeAsync` is because the async methods add nothing.

The documentation for `AddAsync` and `AddRangeAsync` both state this remark:

>This method is async only to allow special value generators, such as the one used by 'Microsoft.EntityFrameworkCore.Metadata.SqlServerValueGenerationStrategy.SequenceHiLo', to access the database asynchronously. For all other cases the non async method should be used.

I don't think the same applies to `SaveChangesAsync` though, and I think generally we prefer to use async methods consistently over non-async methods, which is why I haven't changed these.

We could do with clarifying this decision with the rest of the team.

Reply:
I'd be surprised if it made any appreciable difference either way - the tests will run a fraction quicker or slower?

I'd be inclined to just forget I ever made a comment :)

Reply:
It's probably going to be negligible on the tests but the benefits of using async methods will apply to `SaveChangesAsync` which is why I've kept it in.